get_authors.rb

```diff
@@ -2,8 +2,30 @@
 # This script gets last 5 authors of a file and writes them at the end of the file.
 
 # Get last 5 authors of a file and their email
-def get_authors(file)
-  `git log -5 --pretty=format:"%an;%ae" #{file}`.split(/[;\n]/)
+module Git
+  class Log
```

Comment:
Hmm, I'm not sure what this nesting (`Git` and `Log`) is for. Would you mind to elaborate?

Reply:
Log class depends on output from Git#log and I do not expect it to be used anywhere else.

Reply:
Of course this might look a bit overengineered for this small case, but the Git::Log is abstraction, i.e. this class can parse the log content and can extract some interesting information.

@jackorp if you are going to provide some description, it would probably make sense to add it into source code. This is kind of comments which should actually be in the source code.

Reply:
Yes, comments/description would be nice.
@jackorp, do not limit yourself to one line :).